internal/campaigns/rewirer_mapping.go

```diff
@@ -0,0 +1,53 @@
+package campaigns
+
+import "github.com/sourcegraph/sourcegraph/internal/api"
+
+// RewirerMapping holds a mapping from a changeset spec to a changeset.
+// If the changeset spec doesn't target any changeset (ie. it's new), ChangesetID is 0.
```

Comment:
See my naming suggestion above, but in any case:

```suggestion
// RewirerMapping maps a ChangesetSpec to a Changeset.
// If the ChangesetSpec doesn't match a Changeset (ie. it describes a to-be-created Changeset), ChangesetID is 0.
```